### PR TITLE
Wrapper script to run simple importer and stats validation

### PR DESCRIPTION
**Pull request by ajaits**
Wrapper script to run stats importer and validations.

### CODE REVIEW

File: simple/stats.sh [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+# Defaults
+OUTPUT_DIR=".data/output"
+USAGE="Script to process stats with simple importer.
+Usgae: $(basename $0) [Options]
+Options:
+  -c <file>   Json config file for stats importer
+  -k <api-key> DataCommons API Key
+  -i <dir>    Input directory to process
+  -o <dir>    Output folder for stats importer. Default: $OUTPUT_DIR
+  -m <customdc|maindc> Mode of operation for simple importer
```
Comment (ajaits) [inline simple/stats.sh:12]: default to maindc which generates schema, csv/tmcf, schema.mcf

File: simple/stats.sh [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+# Script to process stats using simple stats loader.
+# Defaults
+OUTPUT_DIR=".data/output"
+USAGE="Script to process stats with simple importer.
+Usgae: $(basename $0) [Options]
+Options:
+  -c <file>   Json config file for stats importer
+  -k <api-key> DataCommons API Key
+  -i <dir>    Input directory to process
+  -o <dir>    Output folder for stats importer. Default: $OUTPUT_DIR
```
Comment (ajaits) [inline simple/stats.sh:11]: add prefix/suffix of mode (customdc or maindc)

File: simple/stats.sh [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+  # Get the tmcf file to validate csv stats.
+  tmcf=$(get_or_generate_tmcf "$OUTPUT_DIR")
+
+  # Run dc-import genmcf
+  cmd="java -jar $DC_IMPORT_JAR genmcf -n 20 -r FULL $OUTPUT_DIR/*.csv $tmcf -o $OUTPUT_DIR/dc_generated"
+  echo_log "Running dc-import validation: $cmd"
+  $cmd >> $LOG 2>&1
+  status="$?"
+  [[ "$status" == "0" ]] || echo_fatal "Failed to run dc-import: $cmd"
+  echo_log "Output of validiton in $OUTPUT_DIR/dc_generated/report.json"
```
Comment (ajaits) [inline simple/stats.sh:185]: typo validation

Comment (ajaits) [reply simple/stats.sh:185]: Done

File: simple/stats.sh (hunk 1 above)
Comment (ajaits) [reply simple/stats.sh:12]: Added TODO for customdc mode

**Review by keyurva — APPROVED** [review]
Thanks Ajai - this is amazing!

**Pull request merged**